contract/tests/test_contract.py

```diff
@@ -279,7 +279,7 @@ def test_last_invoice_post_paid(self):
         )
         self.contract.recurring_create_invoice()
         self.assertEqual(
-            self.acct_line.recurring_next_date, to_date('2018-04-01')
```

Comment:
The current version assure that a customer is billed the same day each month.

Assume this example:
- date start 01/01
- date end 15/03

the customer will receive an invoices at:
* for pre-paid:  01/01, 01/02 and 01/03
* for monthlylastday:  31/01, 28/02 and 31/03
* for post-paid: 01/02, 01/03 and 01/04

for me it make more sens to say post-paid mode invoice date is the last day of the invoicing period (or the last day of the invoicing period + one day).

Let make a RFC and see what other think about this change